stubs/python-crontab/crontab.pyi

```diff
@@ -0,0 +1,233 @@
+import re
+from _typeshed import Incomplete
+from collections import OrderedDict
+from collections.abc import Callable, Generator
+from logging import Logger
+from types import TracebackType
+from typing import Any
+
+__pkgname__: str
+ITEMREX: re.Pattern[str]
+SPECREX: re.Pattern[str]
+DEVNULL: str
+WEEK_ENUM: list[str]
+MONTH_ENUM: list[str]
+SPECIALS: dict[str, str]
+SPECIAL_IGNORE: list[str]
+S_INFO: list[dict[str, Any]]
+PY3: bool
+WINOS: bool
+POSIX: bool
+SYSTEMV: bool
+ZERO_PAD: bool
+LOG: Logger
+CRON_COMMAND: str
+SHELL: str
+current_user: Callable[[], str | None]
+unicode = str
+basestring = str
+
+def open_pipe(cmd: Incomplete, *args: Incomplete, **flags: Incomplete) -> Incomplete: ...
+
+class CronTab:
+    lines: Incomplete
+    crons: Incomplete
+    filen: Incomplete
+    cron_command: Incomplete
+    env: Incomplete
+    root: Incomplete
+    intab: Incomplete
+    def __init__(
+        self, user: str | None = ..., tab: Incomplete | None = ..., tabfile: Incomplete | None = ..., log: Incomplete | None = ...
+    ) -> None: ...
+    def __enter__(self) -> Incomplete: ...
+    def __exit__(
+        self, exc_type: type[BaseException] | None, exc_val: BaseException | None, exc_tb: TracebackType | None
+    ) -> None: ...
+    @property
+    def log(self) -> Incomplete: ...
+    @property
+    def user(self) -> str | None: ...
+    @property
+    def user_opt(self) -> Incomplete: ...
+    def __setattr__(self, name: str, value: Any) -> None: ...
```

Comment:
This only allows setting two specific attributes. I think it's better omitted, because if we keep it typecheckers will allow setting arbitrary attributes.
```suggestion
```